platforms/native/language-native/src/integTest/groovy/org/gradle/language/swift/SwiftIncrementalBuildIntegrationTest.groovy

```diff
@@ -265,11 +262,11 @@ class SwiftIncrementalBuildIntegrationTest extends AbstractInstalledToolChainInt
         }
 
         then:
-        file("app/build/obj/main/debug").assertHasDescendants(expectedIntermediateDescendants(app.application.original))
+        file("app/build/obj/main/debug").assertContainsDescendants(expectedIntermediateDescendants(app.application.original))
```

Comment:
Different versions of `swiftc` generate a bunch of version specific auxiliary files. Ignore these, and just check that the object files are present.